source/js/diva.js

```diff
@@ -939,50 +939,30 @@ module.exports = diva;
         };
 
         /*
-            Given a pageX and pageY value (as could be retreived from a jQuery event object),
```

Comment:
That comment wasn't great, but it does make it more clear that pageX and pageY are relative to the HTML document, rather than the pages of the document in Diva.